com.microsoft.mrtk.standardassets/Materials/MRTK_Standard_Yellow.mat

```diff
@@ -2,15 +2,21 @@
 %TAG !u! tag:unity3d.com,2011:
 --- !u!21 &2100000
 Material:
-  serializedVersion: 6
+  serializedVersion: 8
   m_ObjectHideFlags: 0
   m_CorrespondingSourceObject: {fileID: 0}
   m_PrefabInstance: {fileID: 0}
   m_PrefabAsset: {fileID: 0}
   m_Name: MRTK_Standard_Yellow
   m_Shader: {fileID: 4800000, guid: c331f6c43a2ef0945864cb668f2653c9, type: 3}
-  m_ShaderKeywords: _BORDER_LIGHT_USES_HOVER_COLOR _DIRECTIONAL_LIGHT _DISABLE_ALBEDO_MAP
-    _LIGHTMAPPING_REALTIME _SPECULAR_HIGHLIGHTS
+  m_ValidKeywords:
+  - _BORDER_LIGHT_USES_HOVER_COLOR
+  - _DIRECTIONAL_LIGHT
+  - _DISABLE_ALBEDO_MAP
+  - _SPECULAR_HIGHLIGHTS
+  - _USE_WORLD_SCALE
+  m_InvalidKeywords:
+  - _LIGHTMAPPING_REALTIME
```

Comment:
Do we want to go through and clean up some of these `m_InvalidKeywords`? Do we know if they're from porting these materials from the MRTK2 shader? Unity's shader? Something else? (CC @Cameron-Micka)

Reply:
Oh wow, making the jump to 2021! Note, this is a one-way street for materials. 2021 authored materials cannot be loaded in 2020.

As for `_LIGHTMAPPING_REALTIME` that is a Unity keyword that we don't use. So, it can be safely removed. Unity 2021 started putting unused keywords (for the current shader) in a separately serialized list.

Reply:
Resolving